src/backend/README.md

```diff
@@ -0,0 +1,188 @@
+# s3gw-ui backend
+
+These are notes for developers. Users should rely on [s3gw documentation][1].
+
+[1]: https://s3gw-docs.readthedocs.io/en/latest/?badge=latest
+
+## Setup
+
+```
+$ python3 -m venv venv
+$ source venv/bin/activate
+$ pip install -r requirements.txt -r requirements-dev.txt
+```
+
+## Running
+
+Before running the backend, the frontend needs to be built. Otherwise an error
+will be thrown. It is recommended to follow the instructions for the frontend,
+but a quick gist of the process is as follows:
+
+```
+$ cd frontend/
+$ npm ci
+$ npx ng build
+```
+
+In reality, the backend only requires the `dist/s3gw-ui/` directory to exist.
+For the lazy, creating said directory should be enough.
+
+Running the backend solely requires running the `s3gw_ui_backend.py` script.
+
+```
+$ python3 ./s3gw_ui_backend.py
```

Comment:
I'm not sure how to tell the dev to change to the correct directory a better way. But IMO it is important to tell them where to run the command.
```suggestion
$ cd ./src
$ python3 ./s3gw_ui_backend.py
```